search/includes/classes/queue/class-cron.php

```diff
@@ -212,29 +212,27 @@ public function sweep_jobs() {
 
 		$this->queue->free_deadlocked_jobs();
 
-		// If job limit has been surpassed, don't schedule any new jobs
-		if ( $this->get_processor_job_count() > self::MAX_PROCESSOR_JOB_COUNT ) {
-			return;
-		}
+		$job_count = $this->get_processor_job_count();
+		while ( ! is_wp_error( $job_count ) && $job_count < self::MAX_PROCESSOR_JOB_COUNT ) {
```

Comment:
Should this be `$job_count <= self::MAX_PROCESSOR_JOB_COUNT` now? If the max is scheduled, I don't believe we'd want to add another one.

Reply:
The logic is changed here a bit but this should result in scheduling 5 jobs if MAX_PROCESSOR_JOB_COUNT = 5.

Now it tries to add jobs if this is true (it used to exits here if $job_count > MAX).

IF current $job_count is 4, 4 < 5 => add one more